composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dvsa/front-end",
   "description": "DVSA front-end assets used within all applications",
-  "version": "1.2.29",
+  "version": "1.2.30",
```

Comment:
Have you rebased? Version is currently 1.2.31

Reply:
No. Will update the composer and package and push back up.